### ISSUE TITLE
Plugin not working past 10.4.3

### ISSUE THREAD

**Issue by telans** [jellyfin/jellyfin-plugin-tmdbboxsets#6]
Using latest jellyfin-git, plugin version 4.0.0

```
[2019-12-19 20:47.977] [ERR] Error processing request
System.MissingMethodException: Method not found: 'Void MediaBrowser.Controller.Entities.InternalItemsQuery.set_OrderBy(System.ValueTuple`2<System.String,MediaBrowser.Model.Entities.SortOrder>[])'.
   at Jellyfin.Plugin.TMDbBoxSets.TMDbBoxSetManager.GetMoviesFromLibrary()
   at Jellyfin.Plugin.TMDbBoxSets.TMDbBoxSetManager.ScanLibrary(IProgress`1 progress)
   at Jellyfin.Plugin.TMDbBoxSets.Api.TMDbCollectionsService.Post(RefreshMetadataRequest request)
   at Emby.Server.Implementations.Services.ServiceExecGeneral.<>c__DisplayClass5_0.<CreateExecFn>b__0(Object service, Object request) in /home/arch/.cache/yay/jellyfin-git/src/jellyfin/Emby.Server.Implementations/Services/ServiceExec.cs:line 209
   at Emby.Server.Implementations.Services.ServiceExecGeneral.Execute(Type serviceType, IRequest request, Object instance, Object requestDto, String requestName) in /home/arch/.cache/yay/jellyfin-git/src/jellyfin/Emby.Server.Implementations/Services/ServiceExec.cs:line 88
   at Emby.Server.Implementations.Services.ServiceController.Execute(HttpListenerHost httpHost, Object requestDto, IRequest req) in /home/arch/.cache/yay/jellyfin-git/src/jellyfin/Emby.Server.Implementations/Services/ServiceController.cs:line 160
   at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost httpHost, IRequest httpReq, HttpResponse httpRes, ILogger logger, CancellationToken cancellationToken) in /home/arch/.cache/yay/jellyfin-git/src/jellyfin/Emby.Server.Implementations/Services/ServiceHandler.cs:line 80
   at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken) in /home/arch/.cache/yay/jellyfin-git/src/jellyfin/Emby.Server.Implementations/HttpServer/HttpListenerHost.cs:line 553
```

**Comment by plessbd** [comment]
I am also getting this error and would be willing to help debug

**Comment by telans** [comment]
It's some change they made in master causing it, 10.4.3 works fine for me

**Comment by dkanada** [comment]
Most likely related to the dependency updates.

**Comment by telans** [comment]
> Most likely related to the dependency updates.

I can do a bisect if you think it'll be helpful

**Comment by dkanada** [comment]
Thanks, but it should be fine. Most of the time it's just required to update a few methods or arguments.

**Comment by Artiume** [comment]
This is still broken as of latest master

**Comment by dkanada** [comment]
That's probably because no one has made any changes to this repository.

**Comment by davispuh** [comment]
Getting this crash

```
mar 10 02:34:50 localhost dotnet[10901]: [02:34:50] [FTL] [13] Main: Unhandled Exception
mar 10 02:34:50 localhost dotnet[10901]: System.MissingMethodException: Method not found: 'Void MediaBrowser.Controller.Entities.InternalItemsQuery.set_OrderBy(System.ValueTuple`2<System.String,MediaBrowser.Model.Entities.SortOrder>[])'.
mar 10 02:34:50 localhost dotnet[10901]:    at Jellyfin.Plugin.TMDbBoxSets.TMDbBoxSetManager.GetMoviesFromLibrary()
mar 10 02:34:50 localhost dotnet[10901]:    at Jellyfin.Plugin.TMDbBoxSets.TMDbBoxSetManager.OnTimerElapsed()
mar 10 02:34:50 localhost dotnet[10901]:    at Jellyfin.Plugin.TMDbBoxSets.TMDbBoxSetManager.<.ctor>b__5_0(Object _)
mar 10 02:34:50 localhost dotnet[10901]:    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
mar 10 02:34:50 localhost dotnet[10901]: --- End of stack trace from previous location where exception was thrown ---
mar 10 02:34:50 localhost dotnet[10901]:    at System.Threading.TimerQueueTimer.CallCallback(Boolean isThreadPool)
mar 10 02:34:50 localhost dotnet[10901]:    at System.Threading.TimerQueueTimer.Fire(Boolean isThreadPool)
mar 10 02:34:50 localhost dotnet[10901]:    at System.Threading.TimerQueue.FireNextTimers()
mar 10 02:34:50 localhost dotnet[10901]: Unhandled exception. System.MissingMethodException: Method not found: 'Void MediaBrowser.Controller.Entities.InternalItemsQuery.set_OrderBy(System.ValueTuple`2<System.String,MediaBrowser.Model.Entities.SortOrder>[])'.
mar 10 02:34:50 localhost dotnet[10901]:    at Jellyfin.Plugin.TMDbBoxSets.TMDbBoxSetManager.GetMoviesFromLibrary()
mar 10 02:34:50 localhost dotnet[10901]:    at Jellyfin.Plugin.TMDbBoxSets.TMDbBoxSetManager.OnTimerElapsed()
mar 10 02:34:50 localhost dotnet[10901]:    at Jellyfin.Plugin.TMDbBoxSets.TMDbBoxSetManager.<.ctor>b__5_0(Object _)
mar 10 02:34:50 localhost dotnet[10901]:    at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
mar 10 02:34:50 localhost dotnet[10901]: --- End of stack trace from previous location where exception was thrown ---
mar 10 02:34:50 localhost dotnet[10901]:    at System.Threading.TimerQueueTimer.CallCallback(Boolean isThreadPool)
mar 10 02:34:50 localhost dotnet[10901]:    at System.Threading.TimerQueueTimer.Fire(Boolean isThreadPool)
mar 10 02:34:50 localhost dotnet[10901]:    at System.Threading.TimerQueue.FireNextTimers()
mar 10 02:34:51 localhost systemd[1]: jellyfin.service: Main process exited, code=killed, status=6/ABRT
mar 10 02:34:52 localhost systemd[1]: jellyfin.service: Failed with result 'signal'.
mar 10 02:34:52 localhost systemd[1]: jellyfin.service: Scheduled restart job, restart counter is at 5.
mar 10 02:34:52 localhost systemd[1]: Stopped Jellyfin Media Server.
```

**Comment by oddstr13** [comment]
Should be fixed in version 5.0.0.0
